src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -145,8 +189,10 @@ string GetPlatformCode()
 #endif
 
         ActiveLicenseFindResult result;
+        readonly Func<DateTime> utcDateTimeProvider;
```

Comment:
I wanted to make clear that this func expects an UTC datetime (e.g. `DateTime.UtcNow`) and the code wouldn't handle a potentially necessary transformation if needed. I was also thinking about making the provider to return a `DateTimeOffset` so it wouldn't really matter as we could always transform correctly to utc, but it would be an additional, unnecessary step as of now (and core handles everything using utc DateTime as of now). Maybe we can name it `utcDateProvider`?